### PR TITLE
FreeBSD Foundation 2022q3 entry

### PR DESCRIPTION
**Pull request by Jehops**

### CODE REVIEW

File: 2022q3/freebsd-foundation.adoc [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+The FreeBSD Foundation sponsors many conferences, events, and summits around the
+globe.  These events can be BSD-related, open source, or technology events
+geared towards underrepresented groups.  We support the FreeBSD-focused events
+to help provide a venue for sharing knowledge, working together on projects, and
+facilitating collaboration between developers and commercial users.  This all
+helps provide a healthy ecosystem.  We support the non-FreeBSD events to promote
+and raise awareness of FreeBSD, to increase the use of FreeBSD in different
+applications, and to recruit more contributors to the Project.  We are
+continuing to attend virtual events and planning the June 2022 Developer Summit.
```
Comment (pauamma) [inline 2022q3/freebsd-foundation.adoc]: Is "June 2022" correct? This is the July-to-September report.

Comment (Jehops) [reply 2022q3/freebsd-foundation.adoc]: No.  That should say the November 2022 Developer Summit.  Will fix now.

Comment (pauamma) [reply 2022q3/freebsd-foundation.adoc]: Never mind. It was fixed in the 2nd commit.

Comment (Jehops) [reply 2022q3/freebsd-foundation.adoc]: Are you looking at what's in the file after the second commit?  It shouldn't still say June.

**Pull request merged**